tutor/resources/styles/components/reference-book/navbar.less

```diff
@@ -1,38 +1,29 @@
 .reference-book {
 
   @navbar-spacing: 15px;
-  .navbar {
-    height: @tutor-navbar-height; // set in global/navbar.less
-    // override responsive collapse when screen is small,
-    // we handle hiding items via media rules instead
-    .navbar-nav,
-    .navbar-nav > li {
-      margin: 0;
-      float: left;
-      &.navbar-right { float: right; }
-    }
-    .ui-brand-logo { margin: @navbar-spacing 0 0 @navbar-spacing; }
-    .container-fluid {
-      padding: 0;
-    }
-    ul.nav > li > a {
-      margin: 0 @navbar-spacing 0 0;
-      padding: 0;
-      line-height: @tutor-navbar-height;
+  .tutor-top-navbar {
+    justify-content: space-between;
+    .left-side-controls, .right-side-controls {
+      flex: 1;
     }
     .menu-toggle {
-      padding-left: @navbar-spacing;
+      padding: 0 @navbar-spacing;
       height: @tutor-navbar-height;
       cursor: pointer;
       svg { margin-top: @navbar-spacing; }
     }
 
+    &.menu-open {
+      .menu-toggle {
+        border-right: 1px solid @tutor-neutral-light;
+        background: @tutor-neutral-lightest;
+      }
+    }
     .section-title {
       font-size: 1.8rem;
       margin: 0 0 0 2rem;
-      padding-top: 14px;
       display: flex;
-      max-width: calc(~"100% - 550px");
+      max-width: calc(~"100vw - 550px");
```

Comment:
Is the tilde/quote really needed? I see it's used a lot on the source, but I also see a lot of `calc` expressions without it.

Reply:
hm, no I don't think it is.  I think the tilde is needed if your using less variables like if the 550px was a `@width` or something.   I'm really looking forward to moving to scss since that seems saner to me.